package.json

```diff
@@ -34,12 +34,13 @@
   },
   "homepage": "https://github.com/reduct/registry",
   "devDependencies": {
-    "@reduct/build-tools": "^1.4.5"
+    "@reduct/build-tools": "^1.5.2",
+    "@reduct/logger": "^1.1.0"
   },
   "reduct": {
     "sourceFolder": "Src/",
     "distributionFolder": "Dist/",
     "entryFile": "Registry.js",
-    "globalPackageName": "registry"
+    "globalPackageName": "reductRegistry"
```

Comment:
`browserify` is capable of generating the respective namespace.

From the docs:

```
When opts.standalone is a non-empty string, a standalone module is created with that name and a umd wrapper. You can use namespaces in the standalone global export using a . in the string name as a separator, for example 'A.B.C'. The global export will be sanitized and camel cased.
```